acct-user/p2p/p2p-0.ebuild

```diff
@@ -0,0 +1,12 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+ACCT_USER_ID=392
+ACCT_USER_GROUPS=( p2p )
+ACCT_USER_HOME=/home/p2p
```

Comment:
This seems super weird to me, but looks like the previous ebuilds do the same.

Reply:
What is it that you find weird? Creating a new user to run a daemon? Isn't that the standard practice?

Reply:
Needing to set /home, is the idea I think.

Reply:
Makes sense, fixed.